.ci.yaml

```diff
@@ -2287,6 +2287,32 @@ targets:
       - engine/**
       - DEPS
 
+  - name: Linux gradle_lock_files_check
```

Comment:
How slow is this test? If the answer is >10m, this might be OK, but if it's a few minutes, I'd rather see it added to an existing shard and avoid spinning up a VM just to do this.